src/project-1.js

```diff
@@ -3,137 +3,180 @@
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
   // code here
+  return num * 10;
 };
 
 const subtractFive = (num) => {
   // return num after subtracting five
   // code here
+  return num - 5;
 };
 
 const areSameLength = (str1, str2) => {
   // return true if the two strings have the same length
   // otherwise return false
   // code here
+  if (str1.length === str2.length) {
```

Comment:
Whenever you have an if statement for returning `true` or `false` you can usually simplify it to something like this:
``` return str1.length === str2.length;```